src/components/mainpanel.tsx

```diff
@@ -1,43 +1,49 @@
 import React, { useEffect, useState } from 'react'
+import clsx from 'clsx'
```

Comment:
I see that this package is used by material-ui but you should add this to package.json as a dependency.

Reply:
I fixed it